general/container-allowed-images/tests/ephemeral_containers_allowed.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+spec:
+  containers:
+  - name: main-container
+    image: allowedrepo/some-image:tag
```

Comment:
Should we be mixing ephemeral and normal container tests?

Reply:
It says in pod.spec that containers is required so should it not be included?

Reply:
Not if it's not necessary for validation. It looks like you're using `gator` for the testing right? Does it actually deploy an object or does it just use the structure as a data input?

 When building test cases, it's best to make them as specific as possible, generally.

Reply:
Ah okay, `gator` doesn't actually deploy the object so in that case I can remove the containers field.

Reply:
Yeah, that's probably for the best! That way we can isolate tests to the parts that are testing. :)